fw/src/mgos_uart.h

```diff
@@ -150,10 +153,16 @@ void mgos_uart_config_set_rx_params(struct mgos_uart_config *cfg,
 
 /*
  * Set Tx params in the provided config structure: buffer size `tx_buf_size`
- * and whether Tx flow control is enabled (`tx_fc_ena`).
+ * and whether Tx flow control is enabled (`tx_fc_ena`)
+ * 
+ * and whether TxEn (TxEnable) pin is enabled for RS485 mode (`rs485_ena`) where
+ * date is transmitted and received via the same twisted pair (half duplex).
```

Comment:
well spotted - yes data.
I've actually made a further improvement so it's probably worth me issuing another PR before you accept this.